webpack.config.js

```diff
@@ -13,16 +13,13 @@ var commonConfig = require('./webpack.common.config');
 var license = fs.readFileSync('lib/license-header.txt', 'utf8');
 
 module.exports = _.extend(commonConfig, {
-  entry: './lib/server/serverIndex.js',
+  entry: '.',
   output: {
     path: path.join(__dirname, 'dist'),
     filename: 'okta-auth-js.min.js',
     library: 'OktaAuth',
     libraryTarget: 'umd'
   },
-  resolve: {
```

Comment:
This is not useful. Webpack chose to use browser field over main by default:
https://github.com/webpack/webpack/blob/52184b897f40c75560b3630e43ca642fcac7e2cf/lib/WebpackOptionsDefaulter.js#L42
https://github.com/webpack/webpack/blob/52184b897f40c75560b3630e43ca642fcac7e2cf/lib/WebpackOptionsDefaulter.js#L73